js_modules/dagster-ui/packages/ui-core/src/assets/AssetCatalogAssetGraph.tsx

```diff
@@ -84,12 +86,15 @@ export const AssetCatalogAssetGraph = React.memo(
           () => ({
             opsQuery: selection,
             pipelineName: '',
-            opNames: [''],
+            opNames,
```

Comment:
what does passing opNames do here?

Reply:
its named opNames for historical reasons and requires an empty string also for historical reasons but this is basically tracking the nodes which have been "selected".